model/project_parser_db.go

```diff
@@ -61,7 +63,8 @@ func ParserProjectUpdateOne(query interface{}, update interface{}) error {
 	)
 }
 
-// UpsertWithConfigNumber inserts project if it DNE, otherwise updates if the config number is correct
+// UpsertWithConfigNumber inserts project if it DNE
```

Comment:
extremely small nit: Don't use line breaks to break up clauses in comments. Add a period at the end of this line.